pandas/tests/frame/test_quantile.py

```diff
@@ -155,6 +143,27 @@ def test_quantile_interpolation(self):
                              index=[.25, .5], columns=['a', 'b', 'c'])
         assert_frame_equal(result, expected)
 
+    def test_quantile_interpolation_datetime(self, datetime_frame):
+        # see gh-10174
+
+        # interpolation = linear (default case)
+        df = datetime_frame
+        q = df.quantile(0.1, axis=0, interpolation='linear')
+        assert q['A'] == np.percentile(df['A'], 10)
+
+    def test_quantile_interpolation_int(self, int_frame):
+        # see gh-10174
+
+        df = int_frame
+        # interpolation = linear (default case)
+        q = df.quantile(0.1)
+        assert q['A'] == np.percentile(df['A'], 10)
+
+        # test with and without interpolation keyword
+        q1 = df.quantile(0.1)
```

Comment:
what's the difference between q and q1?

Reply:
They are the same. q1 removed.

Reply:
probably it's best to leave it as is at the moment. could add a TODO though.